sway-lib-std/src/hash.sw

```diff
@@ -1,5 +1,7 @@
 library hash;
 
+use ::constants::ZERO;
+
 // Should this be a trait eventually? Do we want to allow people to customize what `!` does?
 // Scala says yes, Rust says perhaps...
 pub fn not(a: bool) -> bool {
```

Comment:
Aside to this PR...but what is `not` doing here 😂. Can you file an issue to investigate?

Reply:
So close...
![image](https://user-images.githubusercontent.com/59666792/169116468-de9df9d2-d248-43b8-88be-299fe6df64a6.png)

Reply:
Issue: #1593